tools/npy_tempita/__init__.py

```diff
@@ -155,7 +155,7 @@ def from_filename(cls, filename, namespace=None, encoding=None,
             c = f.read()
         if encoding:
             c = c.decode(encoding)
-        elif PY3:
+        else:
             c = c.decode('latin-1')
```

Comment:
In the original CL this wasn't nested under "else:" but I believe that was a mistake